### PR TITLE
Add the latest static Irish translations

### PR DESCRIPTION
**Pull request by petechd**
## What is the context of this PR?
Adds the latest fixes from crowdin to the Irish static messages file.

## How to review
Check the translated strings match crowdin.

## Checklist
 New static content marked up for translation
 Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by petechd** [comment]
[Load Irish PO files for NISRA](https://trello.com/c/N49polb2/4415-load-irish-po-files-for-nisra)

File: app/translations/ga/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -37,157 +38,164 @@ msgstr[2] "%(num)s mí"
...
-msgstr "Ní rogha bhailí í seo"
+#: app/forms/error_messages.py:11 app/forms/error_messages.py:12
+#: app/forms/error_messages.py:13 app/forms/error_messages.py:14
+msgid "Enter an answer"
+msgstr "Iontráil freagra"
+
+#: app/forms/error_messages.py:15
+#, python-format
+msgid "Select an answer <span class=\"u-vh\">to ‘%(question_title)s’</span>"
+msgstr "Roghnaigh freagra<span class=\"u-vh\">ar ‘%(question_title)s’</span>"
```
Comment (berroar) [inline app/translations/ga/LC_MESSAGES/messages.po:59]: Is there supposed to be a space before `<span`?

File: app/translations/ga/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -37,157 +38,164 @@ msgstr[2] "%(num)s mí"
...
-msgid "Census 2021"
-msgstr "Daonáireamh 2021"
-
-#: app/helpers/template_helper.py:32
-msgid "Northern Ireland Statistics and Research Agency logo"
-msgstr "Lógó na Gníomhaireachta um Staidreamh agus Thaighde Thuaisceart Éireann"
+#: app/forms/error_messages.py:19
+#, python-format
+msgid "Select at least one answer <span class=\"u-vh\">to ‘%(question_title)s’</span>"
+msgstr "Roghnaigh freagra amháin ar a laghad<span class=\"u-vh\">ar ‘%(question_title)s’</span>"
```
Comment (berroar) [inline app/translations/ga/LC_MESSAGES/messages.po:69]: Is there supposed to be a space before <span?

File: app/translations/ga/LC_MESSAGES/messages.po (hunk 1 above)
Comment (petechd) [reply app/translations/ga/LC_MESSAGES/messages.po]: It's fixed in crowdin now and added to this PR, thanks!

File: app/translations/ga/LC_MESSAGES/messages.po (hunk 2 above)
Comment (petechd) [reply app/translations/ga/LC_MESSAGES/messages.po]: It's fixed in crowdin now and added to this PR, thanks!

File: app/translations/ga/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -197,944 +205,1121 @@ msgstr[2] "%(num)s lá"
...
+#: templates/confirmation-email-sent.html:29
+msgid "Didn't receive an email?"
+msgstr "Ní bhfuair tú r-phost?"
 
-#: templates/partials/answers/date.html:23
-msgid "Day"
-msgstr "Lá"
+#: templates/confirmation-email-sent.html:30
+msgid "It can take a few minutes for the email to arrive. If it doesn't arrive, check your junk mail, or you can <a href='{send_confirmation_email_url}' id='send-another-email'>send another confirmation email</a>."
+msgstr "Glacann sé cúpla bomaite ar an r-phost teacht chugat. Mura dtagann sé, seiceáil do dhramhphost, nó tig leat <a href='{send_confirmation_email_url}' id='send-another-email'> r-phost dearbhaithe eile a sheoladh</a>."
```
Comment (ajmaddaford) [inline app/translations/ga/LC_MESSAGES/messages.po:720]:
```suggestion
msgstr "Glacann sé cúpla bomaite ar an r-phost teacht chugat. Mura dtagann sé, seiceáil do dhramhphost, nó tig leat <a href='{send_confirmation_email_url}' id='send-another-email'>r-phost dearbhaithe eile a sheoladh</a>."
```

File: app/translations/ga/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -197,944 +205,1121 @@ msgstr[2] "%(num)s lá"
...
+#: templates/errors/404.html:8
+msgid "If you pasted the web address, check you copied the whole address."
+msgstr "Má ghreamaigh tú an seoladh idirlín isteach, déan seiceáil gur chópáil tú an seoladh iomlán isteach."
 
-#: templates/static/accessibility.html:58
-msgid "Reporting accessibility problems"
-msgstr ""
+#: templates/errors/404.html:9
+msgid "If the web address is correct or you selected a link or button, <a href='{url}'>contact us</a> for more help."
+msgstr "Má tá an seoladh idirlín ceart nó má roghnaigh tú nasc nó cnaipe, <a href='{url}'> déan teagmháil linn </a> do thuilleadh cuidithe."
```
Comment (ajmaddaford) [inline app/translations/ga/LC_MESSAGES/messages.po:903]:
```suggestion
msgstr "Má tá an seoladh idirlín ceart nó má roghnaigh tú nasc nó cnaipe, <a href='{url}'>déan teagmháil linn</a> do thuilleadh cuidithe."
```

File: app/translations/ga/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -197,944 +205,1121 @@ msgstr[2] "%(num)s lá"
...
+#: templates/errors/500.html:8
+msgid "If you have started a survey, your answers have been saved."
+msgstr "Má thosaigh tú ar shuirbhé, sábháladh do fhreagraí."
 
-#: templates/static/accessibility.html:68
-msgid "<a href='{url}'>Equalities Commission for Northern Ireland (ECNI)</a> if you live in Northern Ireland"
-msgstr ""
+#: templates/errors/500.html:9
+msgid "<a href='{url}'>Contact us</a> if you need to speak to someone about your survey."
+msgstr "<a href='{url}'> Déan teagmháil linn </a> más gá duit labhairt le duine éigin faoi do shuirbhé."
```
Comment (ajmaddaford) [inline app/translations/ga/LC_MESSAGES/messages.po:923]:
```suggestion
msgstr "<a href='{url}'>Déan teagmháil linn</a> más gá duit labhairt le duine éigin faoi do shuirbhé."
```

File: app/translations/ga/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -197,944 +205,1121 @@ msgstr[2] "%(num)s lá"
...
+#: templates/layouts/_base.html:61
+msgid "Tell us whether you accept cookies"
+msgstr "Abair linn an nglacann tú fianáin"
 
-#: templates/static/accessibility.html:119
-msgid "user testing with the public on different devices, different locations, in their homes with various assistive technologies including free and paid versions"
-msgstr ""
+#: templates/layouts/_base.html:62
+msgid "We use <a href='{cookie_settings_url}'>cookies to collect information</a> about how you use census.gov.uk. We use this information to make the website work as well as possible and improve our services."
+msgstr "Úsáidimid <a href='{cookie_settings_url}'>fianáin le heolas a bhailiú</a>faoin úsáid a bhaineann tú as census.gov.uk. Úsáidimid an t-eolas seo le go n-oibreoidh an suíomh gréasáin chomh maith agus is féidir agus lenár seirbhísí a fheabhsú."
```
Comment (ajmaddaford) [inline app/translations/ga/LC_MESSAGES/messages.po:1023]:
```suggestion
msgstr "Úsáidimid <a href='{cookie_settings_url}'>fianáin le heolas a bhailiú</a> faoin úsáid a bhaineann tú as census.gov.uk. Úsáidimid an t-eolas seo le go n-oibreoidh an suíomh gréasáin chomh maith agus is féidir agus lenár seirbhísí a fheabhsú."
```

**Review by ajmaddaford — COMMENTED** [review]
There are some minor issues with missing/extra spaces.

File: app/translations/ga/LC_MESSAGES/messages.po [hunk 7]
```diff
@@ -197,944 +205,1121 @@ msgstr[2] "%(num)s lá"
...
+msgid "If you need to give more feedback, please <a href='{contact_us_url}'>contact us</a>."
+msgstr "Más gá duit níos mó aiseolais a thabhairt, déan <a href='{contact_us_url}'>teagmháil linn</a>."
+
+#: app/routes/errors.py:161
+msgid "Sorry, there was a problem sending the access code"
+msgstr "Ár leithscéal, bhí fadhb leis an chód rochtana a sheoladh"
+
+#: app/routes/errors.py:167
+msgid "You can try to <a href='{retry_url}'>request a new access code again</a>."
+msgstr "Is féidir triail a bhaint as <a href='{retry_url}'> cód úr rochtana a iarraidh arís</a>."
```
Comment (MebinAbraham) [inline app/translations/ga/LC_MESSAGES/messages.po:318]:
```suggestion
msgstr "Is féidir triail a bhaint as <a href='{retry_url}'>cód úr rochtana a iarraidh arís</a>."
```

File: app/translations/ga/LC_MESSAGES/messages.po [hunk 8]
```diff
@@ -197,944 +205,1121 @@ msgstr[2] "%(num)s lá"
...
+msgid "If this problem keeps happening, please <a href='{contact_us_url}'>contact us</a> for help."
+msgstr "Má leantar den fhadhb, déan <a href='{contact_us_url}'>teagmháil linn</a> le cuidiú a iarraidh."
+
+#: app/routes/errors.py:189
+msgid "Sorry, there was a problem sending the confirmation email"
+msgstr "Ár keithscéal, bhí fadhb leis an r-phost dearbhaithe a sheoladh"
+
+#: app/routes/errors.py:190
+msgid "You can try to <a href='{retry_url}'>send the email again</a>."
+msgstr "Is féidir triail a bhaint as <a href='{retry_url}'> an r-phost a sheoladh arís</a>."
```
Comment (MebinAbraham) [inline app/translations/ga/LC_MESSAGES/messages.po:330]:
```suggestion
msgstr "Is féidir triail a bhaint as <a href='{retry_url}'>an r-phost a sheoladh arís</a>."
```

File: app/translations/ga/LC_MESSAGES/messages.po [hunk 9]
```diff
@@ -197,944 +205,1121 @@ msgstr[2] "%(num)s lá"
...
+
+#: app/routes/errors.py:211 templates/errors/403.html:3
+#: templates/errors/403.html:6 templates/errors/submission-failed.html:5
+#: templates/errors/submission-failed.html:8
+msgid "Sorry, there is a problem"
+msgstr "Tá brón orm, tá fadhb ann"
+
+#: app/routes/errors.py:212
+msgid "You can try to <a href='{retry_url}'>submit your feedback again</a>."
+msgstr "Is féidir triail a bhaint as <a href='{retry_url}'> d'aiseolas a chur isteach arís</a>."
```
Comment (MebinAbraham) [inline app/translations/ga/LC_MESSAGES/messages.po:340]:
```suggestion
msgstr "Is féidir triail a bhaint as <a href='{retry_url}'>d'aiseolas a chur isteach arís</a>."
```

File: app/translations/ga/LC_MESSAGES/messages.po [hunk 10]
```diff
@@ -197,944 +205,1121 @@ msgstr[2] "%(num)s lá"
...
+#: templates/thank-you.html:26
+msgid "Your answers were submitted for <span>{ru_name}</span> on {submitted_date_time}"
+msgstr "Cuireadh do fhreagraí isteach do <span>{ru_name}</span> ar {submitted_date_time}"
 
-#: templates/static/accessibility.html:41
-msgid "the online system for getting a new census code has not been fully tested so parts of it may not be accessible to all users"
-msgstr ""
+#: templates/thank-you.html:32
+msgid "Transaction ID: <b>{transaction_id}</b>"
+msgstr "ID Idirbheart:<b>{transaction_id}</b>"
```
Comment (MebinAbraham) [inline app/translations/ga/LC_MESSAGES/messages.po:863]:
```suggestion
msgstr "ID Idirbheart: <b>{transaction_id}</b>"
```

File: app/translations/ga/LC_MESSAGES/messages.po [hunk 11]
```diff
@@ -197,944 +205,1121 @@ msgstr[2] "%(num)s lá"
...
+#: templates/partials/feedback-call-to-action.html:9
+msgid "Give feedback"
+msgstr "Tabhair aiseolas"
 
-#: templates/static/privacy.html:10
-msgid "The information you provide when you fill in your census rehearsal questionnaire is protected by law."
-msgstr "Beidh an t-eolas a thabharfaidh tú dúinn nuair a líonfaidh tú an réamhchleachtadh de cheistiúchán daonáirimh isteach á chosaint ag an dlí."
+#: templates/partials/individual-response-guidance.html:24
+msgid "You can <em>share your household access code</em> with the people you live with so they can complete their own sections."
+msgstr "Tig leat <em>do chód rochtana teaghlaigh a roinnt</em>leis na daoine a gcónaíonn tú leo sa dóigh go dtig leo a míreanna féin a chomhlánú."
```
Comment (MebinAbraham) [inline app/translations/ga/LC_MESSAGES/messages.po:1095]:
```suggestion
msgstr "Tig leat <em>do chód rochtana teaghlaigh a roinnt</em> leis na daoine a gcónaíonn tú leo sa dóigh go dtig leo a míreanna féin a chomhlánú."
```

File: app/translations/ga/LC_MESSAGES/messages.po [hunk 12]
```diff
@@ -197,944 +205,1121 @@ msgstr[2] "%(num)s lá"
...
+msgid "This is the last viewed question in this section"
+msgstr "Is í seo an cheist bhreathnaithe dheireanach sa mhír seo"
 
-#: templates/static/privacy.html:16
-msgid "We collect information from your questionnaire under our statutory objective to promote and safeguard the production of official statistics that serve the public good. This means that any personal data we collect will only ever be used to produce statistics or undertake statistical research. People will not be identified within those statistics or research."
-msgstr "Bailíonn muid eolas ó do cheistiúchán de réir ár gcuspóra reachtúil le staidreamh oifigiúil ar mhaithe leis an ghnáthphobal a dhéanamh go forásach agus go slán sábháilte. Ciallaíonn sé seo nach n-úsáidfear cibé eolas a bhaileoidh muid ach le staidreamh nó le taighde staidrimh a dhéanamh. Ní aithneofar daoine taobh istigh den staidreamh nó den taighde sin."
+#: templates/partials/last_viewed_question_guidance.html:10
+msgid "You can also <a id='section-start-link' href='{url}'>go back to the start of the section</a>"
+msgstr "Is féidir fosta <a id='section-start-link' href='{url}'> dul ar ais chuig tús na míre</a>"
```
Comment (MebinAbraham) [inline app/translations/ga/LC_MESSAGES/messages.po:1107]:
```suggestion
msgstr "Is féidir fosta <a id='section-start-link' href='{url}'>dul ar ais chuig tús na míre</a>"
```

**Comment by petechd** [comment]
Spacing issues fixed. Thanks guys!

**Review by MebinAbraham — APPROVED** [review]

**Review by berroar — APPROVED** [review]

**Pull request merged**